hack/otel/usage.md

```diff
@@ -0,0 +1,25 @@
+# Sample stack for testing OTEL functionality with the CLI
+
+To easily test the OTEL functionality present in the CLI, you can spin up a small demo compose stack that includes:
```

Comment:
It's good to avoid wording like "simple", "easy"; what may look simple to you / us, may not be simple to someone reading.

```suggestion
To test the OTEL functionality present in the CLI, you can spin up a demo compose stack that includes:
```

Reply:
Fair enough, will change